sci-biology/bowtie/files/bowtie-1.1.2-tbb-tinythread-missing.patch

```diff
@@ -0,0 +1,14 @@
+--- bowtie-1.1.2/Makefile
```

Comment:
the patch should have a proper description, even if it is short, see https://dev.gentoo.org/~vapier/clean-patches